python/dgl/distributed/sparse_emb.py

```diff
@@ -58,148 +60,101 @@ class DistEmbedding:
 
     Note
     ----
-    When a ``DistEmbedding``  object is used when the deep learning framework is recording
-    the forward computation, users have to invoke py:meth:`~dgl.distributed.SparseAdagrad.step`
+    When a ``NodeEmbedding``  object is used when the deep learning framework is recording
+    the forward computation, users have to invoke py:meth:`~dgl.distributed.optim.SparseAdagrad.step`
     afterwards. Otherwise, there will be some memory leak.
     '''
     def __init__(self, num_embeddings, embedding_dim, name=None,
                  init_func=None, part_policy=None):
         self._tensor = DistTensor((num_embeddings, embedding_dim), F.float32, name,
-                                  init_func, part_policy)
+                                  init_func=init_func)
```

Comment:
we don't need to provide partition policy? how does the distributed tensor know which partition policy to use?